File: PB - More Exercises Python/7.0 Nested Loops - More Exercises/08. Secret Door's Lock.py
hundred_num, double_num, single_num = [int(input()) + 1 for _ in range(3)]
prime_nums = [2, 3, 5, 7]

for h in range(2, hundred_num, 2):
    for d in range(2, double_num):
        for s in range(2, single_num, 2):
            if d in prime_nums:
                print(f'{h} {d} {s}')


'''
3
5
5
'''

File: PB - More Exercises GO/7.0 Nested Loops - More Exercises/08. Secret Door's Lock.go
package main

import "fmt"

func main() {
	var hundredNum, doubleNum, singleNum int
	fmt.Scan(&hundredNum, &doubleNum, &singleNum)
	for h := 2; h <= hundredNum; h += 2 {
		for d := 2; d <= doubleNum; d++ {
			for s := 2; s <= singleNum; s += 2 {
				if d == 2 || d == 3 || d == 5 || d == 7 {
					fmt.Println(h, d, s)
				}
			}
		}
	}
}

/*
name   :08. Secret Door's Lock
input  :2 5 5
output :
2 2 2
2 2 4
2 3 2
2 3 4
2 5 2
2 5 4
*/
